src/Components/Aspire.Milvus.Client/Aspire.Milvus.Client.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>$(ComponentCommonPackageTags) milvus</PackageTags>
```

Comment:
Do we also want `database vector search` here? That's what https://www.nuget.org/packages/Qdrant.Client has. 

Note https://www.nuget.org/packages/Milvus.Client just has `milvus vector database`.

Reply:
We didn't add those tags to the Aspire components, could add them no problem